frontend/integration-tests/protractor.conf.ts

```diff
@@ -33,7 +33,7 @@ const htmlReporter = new HtmlScreenshotReporter({
   dest: `./${screenshotsDir}`,
   inlineImages: true,
   captureOnlyFailedSpecs: true,
-  filename: 'test-gui-report.html',
+  filename: 'test-gui-protractor-report.html',
```

Comment:
Maybe just `protractor-report.html`

Reply:
Hi, had to trap for exit and then run the report generator, should be all set now.
Renamed to `protractor-report.html`.